### PR TITLE
Added a React Login Page

### PR DESCRIPTION
**Pull request by harshchandwani**

### CODE REVIEW

**Review by MOHAMED-EHAB-DEV — CHANGES_REQUESTED** [review]
Please add A README File and read this guide to create a readme file: https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md + Some Screenshots to see the app

**Comment by mudit023** [comment]
Hey @harshchandwani, thanks for the contribution. Please follow our contribution and [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md). It's mandatory for all contributors to follow the guidelines.

File: login-page-react/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Login Page Using ReactJS
+
```
Comment (mudit023) [inline login-page-react/README.md:2]: Create a readme file according to our [template guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md}

**Review by mudit023 — CHANGES_REQUESTED** [review]
- [ ] Please remove unnecessary files, photos, and logos.
- [ ] Add a proper readme file like this [https://github.com/mudit023/templates/tree/main/animated-newsletter#readme]
- [ ] Make use of CSS variables where ever possible.

**Pull request closed without merging**